brvs-core/src/main/java/iroha/validation/utils/ValidationUtils.java

```diff
@@ -30,7 +30,7 @@
 
 public interface ValidationUtils {
 
-  static final JsonParser parser = new JsonParser();
+  JsonParser parser = new JsonParser();
```

Comment:
why?

Reply:
it was unnecessary since interfaces contain only static final fields

Reply:
oh, yeah. cool then.